tool/tctl/common/app_command.go

```diff
@@ -104,6 +104,10 @@ func (c *AppsCommand) ListApps(ctx context.Context, clt auth.ClientI) error {
 		}
 	}
 
+	if servers == nil {
```

Comment:
Same as above.

Reply:
As we do a single assignment for `	coll := &appServerCollection{servers: servers}` it seems best to keep like this. Otherwise we have to have the assignment in each case. And the if check, assign to empty list for the non text formts.

Reply:
These have been removed when not needed. Please see updates